lib/players/human.rb

```diff
@@ -0,0 +1,9 @@
+class Player::Human < Player
+
+  def move(board)
+    puts "Please choose a move, 1 - 9."
+    puts "The cell numbers go top to bottom, left to right."
+    move = gets.chomp
```

Comment:
You don't need to store your return value in a local variable; just utilize the implicit return value of the expression.
```
def move(board)
  puts "Please choose a move, 1 - 9."
  puts "The cell numbers go top to bottom, left to right."
  gets.chomp
end
```